### ISSUE TITLE
FNX-3308 ⁃ [Bug] The Font Size pointer is cutout when selecting the 50% in the Accessibility menu

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#10138]
## Steps to reproduce
1. Launch Fenix.
1. Reach Settings > Accessibility > Disable Automatic Font.
1. Set the Font Size to 50%.
1. Observe the Font Size pointer.
    1. Expected behavior
The Font Size pointer should be fully displayed.
    1. Actual behavior
The Font Size pointer is cutout when selecting the 50% in the Accessibility menu
    1. Device information

* Android device: **Google Pixel 4 XL (10).**
* Fenix version: **Nightly 4/23.**

![Screenshot](https://user-images.githubusercontent.com/42831109/80082039-6ff47a80-855c-11ea-8f33-10e8d07cf67b.png)UNITO-UNDERSCORE!20200423-121559!

**Comment by cadeyrn** [comment]
Duplicate of #10083.

**Comment by abodea** [comment]
Thank you @cadeyrn.
